source/_integrations/reolink.markdown

```
@@ -476,6 +476,101 @@ For example, you can place arrow buttons on the card to [control the camera](/da
 
 You can receive rich notifications on your phone when someone rings a Reolink doorbell or a Reolink camera detects an event like motion or a person.
 
+![afbeelding](https://github.com/user-attachments/assets/2f58667a-9799-44f6-8cd4-c2ea7c993ba7)
+
+![afbeelding](https://github.com/user-attachments/assets/3bbb0d6c-24ea-42aa-a557-cf87292f8830)
+
+{% details "Rich notification tutorial" icon="mdi:cursor-hand" %}
+
+Prerequisites:
+
+- This [Reolink integration](#configuration)
```

Markdownlint (0.35.0)

Link fragments should be valid

(MD051, link-fragments)